plugins/callback/anstomlog.py

```diff
@@ -0,0 +1,395 @@
+# coding=utf-8
```

Comment:
Where does this plugin come from? What is the license attached to this code?

Reply:
it from https://github.com/octplane/ansible_stdout_compact_logger